EmployeeManagementService/EmployeeManagementService.Infrastructure/Persistence/EmployeeRepository.cs

```diff
@@ -17,5 +15,24 @@ public async Task<List<Employee>> GetAllEmployees()
                 return await context.Employees.Select(e => new Employee() { FirstName  = e.FirstName, LastName = e.LastName, Role = e.Role, Username = e.Username, Active = e.Active }).ToListAsync();
             }
         }
+
+        public async Task CreateEmployee( string firstName, string lastName, string username, byte[] password, string role, bool active = true)

```

Comment:
right direction. assigning it default in parameter is good.

but now, look at how you are instantiating the employee. does it even take that field?